### ISSUE TITLE
Adapt input text window to the inserted phrase length

### ISSUE THREAD

**Issue by le22didbe** [wikitongues/poly#262]
When inserting a long phrase in source/target language, input window's size remains fixed, not showing full inserted phrase. Window's size should adapt to text length.

**Comment by FredericoAndrade** [comment]
Quick solution: https://github.com/andreypopp/react-textarea-autosize

**Comment by AmrAdelKhalil** [comment]
I don't think that making source/target language input not fixed will be something useful, Instead if we make inserting new phrase input window not fixed will be much better.
